### PR TITLE
initial Readme updates for OSDeploy

### PR DESCRIPTION
**Pull request by dprince**
Some required README updates now that we have the new
deployment workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -610,37 +612,35 @@ Create a base RHEL data volume prior to deploying OpenStack.  This will be used
...
+    If you write the above YAML into a file called deploy.yaml you can create the OpenStackDeploy via this command:
 
-    # run ansible driven OpenStack deployment
-    ./tripleo-deploy.sh -p
+    ```bash
+    oc create -f deploy.yaml
     ```
 
+    As the deployment runs it will create a Kubernetes Job to execute the Ansible playbooks. You can
+    tail the logs of this job/pod to watch the Ansible playbooks run.
```
Comment (stuggi) [inline README.md:642]: should we also describe how to run it manually via the openstackclient for debugging after a deployment might have failed?

File: README.md [hunk 2]
```diff
@@ -741,43 +741,18 @@ Once you customize the above template/examples for your environment, create/upda
...
 ## Run the software deployment
 
-* Wait for the OpenStackConfigGenerator to finish the playbook rendering job
+* Wait for the OpenStackConfigGenerator to finish the playbook rendering job.
 
-* In the openstackclient pod
+* Obtain the hash/digest of the latest OpenStackConfigVersion.
 
-a) check for new version of rendered playbooks and accept them
+* Install the pre-requisites on overcloud systems for ceph-ansible
```
Comment (stuggi) [inline README.md:748]: we can remove this step as it is no longer required. lvm2 is now part of the ooo bootstrap package list.

File: README.md (hunk 1 above)
Comment (dprince) [reply README.md]: good call, just pushed a new commit with this in it.

File: README.md (hunk 2 above)
Comment (dprince) [reply README.md]: done

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**